base/stream.jl

```diff
@@ -348,8 +348,29 @@ if Sys.iswindows()
     ispty(s::IO) = false
 end
 
-"    displaysize(io) -> (lines, columns)
-Return the nominal size of the screen that may be used for rendering output to this io object"
+"""
+    displaysize(io) -> (lines, columns)
```

Comment:
Might be worth putting `io` in brackets here, as we typically use that in documentation to denote an optional positional argument and `displaysize` has a 0-arg method. It would also be good to describe what the behavior of `displaysize()` is (i.e. the 0-arg method).